client/editSchedule.js

```diff
@@ -160,6 +151,16 @@ Template.editEventModal.helpers({
   formType: function() {
     return this._id ? "update" : "add";
   },
+  multipleDayCompetition: function() {
+    return Competitions.findOne(this.competitionId).numberOfDays > 1;
+  },
+  nthDayOptions: function() {
+    var options = [];
+    for(var i = 0; i < Competitions.findOne(this.competitionId).numberOfDays; i += 1) {
```

Comment:
You go Lars! That's an excellent way of putting it - "Ruby/Python doesn't have ++, so I decided it doesn't exist".